config/auth.php

```diff
@@ -104,9 +100,9 @@
     'passwords' => [
         'users' => [
             'provider' => 'users',
-            'email' => 'auth.emails.password',
```

Comment:
Did this part just not apply?

Reply:
It was an old 5.2 vestige that was never removed: <https://github.com/laravel/laravel/blob/5.2/config/auth.php>

Reply:
a-ha!